file_formats/ie_formats/ini_anim.htm

```diff
@@ -0,0 +1,849 @@
+        <li class="anchors"><strong>color_chunks = int</strong><br /><span class="unknown">It is likely that this attribute expects the color gradient index for the associated chunk animation.</span></li>
+        <li class="anchors"><strong>sound_freq = int</strong><br /><span class="unknown">It is likely that this attribute specifies the delay between creature sounds.</span></li>
+        <li class="anchors"><strong>sound_death = resref</strong><br /><span class="unknown">It is likely that this attribute specifies the sound file played back when dying.</span></li>
+        <li class="anchors"><strong>personal_space = int</strong><br />This attribute expects the diameter of the personal space the creature takes up on the world map. It is measured in search map units. The default diameter for humanoid creatures is 3.</li>
+        <li class="anchors"><strong>cast_frame = int</strong><br /><span class="unknown">It is likely that this attribute specifies the sound delay (in animation frames) for casting animations.</span></li>
+        <li class="anchors"><strong>height_offset = int</strong><br />This attribute expects a numeric value that is used to calculate the bounding box for effects, health bar and other visual markers.</li>
+        <li class="anchors"><strong>brightest = boolean</strong><br />This attribute defines a blending mode when the sprite is rendered onto the screen. If set without the "multiply_blend" attribute then <span class="code">GL_ONE_MINUS_DST_COLOR</span> blending is performed. If both "brightest" and "multiply_blend" are set then <span class="code">GL_SRC_COLOR</span> blending is performed.</li>
+        <li class="anchors"><strong>multiply_blend = boolean</strong><br />This attribute defines a blending mode when the sprite is rendered onto the screen. If set without the "brightest" attribute then <span class="code">GL_DST_COLOR</span> blending is performed. If both "brightest" and "multiply_blend" are set then <span class="code">GL_SRC_COLOR</span> blending is performed.</li>
+        <li class="anchors"><strong>light_source = boolean</strong><br />When this attribute is set then environmental lighting such as day/night, weather or shadows do not affect the creature animation.</li>
+        <li class="anchors"><strong>new_palette = resref</strong><br />This attribute applies a replacement palette from the specified BMP resource to the creature animation.</li>
+        <li class="anchors"><strong>sound_ref = resref</strong><br /><span class="unknown">The meaning of this attribute is unknown.</span></li>
+        <li class="anchors"><strong>combat_round_0 = resref<br />combat_round_1 = resref<br />combat_round_2 = resref<br />combat_round_3 = resref<br />combat_round_4 = resref</strong><br /><span class="unknown">It is likely that these attributes define BMP resources that are repurposed for controlling how attack animation sequences are triggered within a combat round.</span></li>
```

Comment:
That would be pretty odd naming then, since the combat bitmaps (rndbase*) are separate for melee apr sets and launchers. Just thinking out loud.

Reply:
They are defined only for a small number of creatures (zombies, ankhegs, iwd goblins and orcs). I don't know how exactly these bitmaps are used, but the content is very similar to the rndbase* bitmaps.